### PR TITLE
Contact Form

### LINKED ISSUE #51: Add contact us form in the contact section.
I want to make the contact us section better by adding a form. Please assign me this issue.

### PR DESCRIPTION
**Pull request by padmajabhol**
I added a contact form as the issue demanded.

Please include a summary of the change and which issue is fixed.
Fixes #51 

## List any dependencies that are required for this change 
none

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
Screenshot (61)

    - After :
Screenshot (60)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -374,6 +374,21 @@ hr {
 
 /*********** footer ***********/
 
+.form-control {
+    margin-bottom: 8px;
+
```
Comment (harikesh409) [inline css/style.css:379]: remove the blank line.

File: index.html [hunk 2]
```diff
@@ -167,13 +167,23 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
-                        <div class="input-group">
+                        <!-- <div class="input-group">
```
Comment (harikesh409) [inline index.html:170]: You can remove the unnecessary code no need to keep comments.

[Yagni](https://martinfowler.com/bliki/Yagni.html)

File: index.html [hunk 3]
```diff
@@ -167,13 +167,23 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
...
                             <input type="email" class="form-control py-3" placeholder="Enter your Email">
-                            <div class="input-group-append">
+                            <!-- <div class="input-group-append">
                                 <button class="input-group-text subscribe-button text-white px-5 font-weight-bold">
                                     Contact Us
                                 </button>
                             </div>
+                             -->
+                        </div> -->
+                        <input type="text" class="form-control py-3" placeholder="Your Name">
```
Comment (harikesh409) [inline index.html:179]: Refer [bootstrap forms](https://getbootstrap.com/docs/4.3/components/forms/) and the forms  divs using `form-group` classes. Also give some margins it looks a bit tight as there is no space between the container background and input fields.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Also make the same changes in `team.html`

**Comment by padmajabhol** [comment]
> Also make the same changes in `team.html`

Sure

**Comment by padmajabhol** [comment]
@harikesh409 Please review the changes of the latest commit Contactform.

**Comment by Khushi-agarwal** [comment]
@harikesh409 can I work on this issue?

File: team.html [hunk 4]
```diff
@@ -214,13 +214,15 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
...
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
-                        <div class="input-group">
-                            <input type="email" class="form-control py-3" placeholder="Enter your Email">
-                            <div class="input-group-append">
-                                <button class="input-group-text subscribe-button text-white px-5 font-weight-bold">
-                                    Contact Us
-                                </button>
-                            </div>
+                        </div>
```
Comment (harikesh409) [inline team.html:217]: unnecessary closing div tag.

File: team.html [hunk 5]
```diff
@@ -237,8 +239,8 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 <div class="col-lg-2 col-md-2 col-sm-6 col-12 offset-lg-1">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Links</h5>
                     <ul>
-                        <a class="text-white mb-lg-3" href="index.html">About</a><br>
-                        <a class="text-white mb-lg-3" href="./team.html">Our Team</a></br>
+                        <a class="text-white mb-lg-3" href="#about">About</a><br>
```
Comment (harikesh409) [inline team.html:242]: The link should be `index.html#about`

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harikesh409** [comment]
> @harikesh409 can I work on this issue?

Already the issue is being worked by @padmajabhol, so choose some other issue.

**Review by harikesh409 — APPROVED** [review]

**Comment by harikesh409** [comment]
moved to #300

**Pull request closed without merging**